src/helpers.c

```diff
@@ -1199,3 +1200,17 @@ char *get_tracking_dir(void)
 {
 	return sys_path_join(globals.state_dir, "bundles");
 }
+
+bool confirm_action(const char *warning_msg, const char *action)
+{
+	int response;
+
+	if (warning_msg) {
+		warn("%s\n", warning_msg);
+	}
+	info("Do you want to continue%s? (y/N): ", action ? action : "");
+	response = tolower(getchar());
+	info("%s\n", response == 'y' ? "y" : "N");
```

Comment:
Why do we need to print that? It seams redundant because getchar() already echoes the character.

Reply:
hmm I didn't see that printed in the tests output I thought it wasn't being printed.

Reply:
but you are right, I just tested that and it is echoed. I'll remove this.